docs/site/reference/markdown.md

```
@@ -0,0 +1,101 @@
+- Block quotes (`>`)
+- Ordered and unordered lists, nested
+- Inline links: `[text](url)` and `[text](url "title")`
+- Inline images: `![alt](url)`
+- Autolinks: `<https://example.com>`, `<user@example.com>`
+- Horizontal rules (`---`, `***`, `___`)
+- Reference-style links: `[text][label]` with `[label]: url` elsewhere in the document
+
+## GitHub Flavored Markdown (GFM)
+
+- **Tables** — pipe-delimited with a separator row:
+  ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)